src/Agent.Worker/Common/FileSystemHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace Agent.Worker.Common
+{
+    public static class FileSystemHelper
+    {
+        public static void WriteToFile(string file, object value)
```

Comment:
It's not just write to file, it serialize the object into JSON then write it to file, maybe change the function name to indicate that, like `JsonSerializeToFile()` ?